src/mame/drivers/c64.cpp

```diff
@@ -5,7 +5,7 @@
     TODO:
 
     - floating bus writes to peripheral registers in m6502.c
-    - sort out kernals between PAL/NTSC
+    - sort out kernels between PAL/NTSC
```

Comment:
C64 documentation refers to the system ROM as the "kernal" - it's not a real word, but it is the name of the Commodore system ROM.